articles/architecture-scenarios/implementation/b2b/b2b-operations.md

```diff
@@ -40,14 +40,28 @@ useCase:
 
 <%= include('../../_includes/_operations/_notifications.md', { platform: 'b2b' }) %>
 
+## Organization Admin Portal
```

Comment:
I can't make this change without changing a bunch of things as this will change the link to this section.  Also, we have another section for organization provisioning.  Provisioning doesn't require a portal, this was specific to the portal.